### PR TITLE
s2488: remove MAQ references in Advanced Search

### PR DESCRIPTION
**Pull request by hjeeves**
+ change connection pool configuration (new non-Docker dev/prod stanza is commented out in the examples context.xml)

### CODE REVIEW

File: caom2-search-server/src/main/java/ca/nrc/cadc/uws/AdvancedRunner.java [hunk 1]
```diff
@@ -135,15 +133,8 @@ public void setSyncOutput(SyncOutput so) {
...
-
-        if (StringUtil.hasText(maqActivatedParam)) {
-            tapServiceURI = ALTERNATE_TAP_SERVICE_URI;
-            tapServiceKey = ALT_TAP_SERVICE_URI_PROPERTY_KEY;
-        } else {
-            tapServiceURI = DEFAULT_TAP_SERVICE_URI;
-            tapServiceKey = TAP_SERVICE_URI_PROPERTY_KEY;
-        }
+        tapServiceURI = DEFAULT_TAP_SERVICE_URI;
+        tapServiceKey = TAP_SERVICE_URI_PROPERTY_KEY;
```
Comment (at88mph) [inline caom2-search-server/src/main/java/ca/nrc/cadc/uws/AdvancedRunner.java]: Make these one liners, please.

File: caom2-search-server/src/main/resources/META-INF/resources/css/caom2_search.css [hunk 2]
```diff
@@ -601,15 +601,15 @@ canvas.aladin-imageCanvas {
...
-.maq-label {
-  vertical-align: bottom;
-  font-weight: normal;
-  font-size: 9pt;
-}
+/*.maq-label {*/
+/*  vertical-align: bottom;*/
+/*  font-weight: normal;*/
+/*  font-size: 9pt;*/
+/*}*/
```
[the remark this answers is not in the document]
Comment (hjeeves) [reply caom2-search-server/src/main/resources/META-INF/resources/css/caom2_search.css]: done

File: caom2-search-server/src/main/resources/META-INF/resources/js/hierarchy.js [hunk 3]
```diff
@@ -144,8 +144,7 @@
                 }
               },
               SYNC_ENDPOINT: '/sync',
-              TAP_MAQ_URI: 'ivo://cadc.nrc.ca/tap/maq',
-              TAP_URI: 'ivo://cadc.nrc.ca/tap',
+              //TAP_URI: 'ivo://cadc.nrc.ca/tap',
```
[the remark this answers is not in the document]
Comment (hjeeves) [reply caom2-search-server/src/main/resources/META-INF/resources/js/hierarchy.js]: done

File: caom2-search-server/examples/default/src/main/webapp/META-INF/context.xml [hunk 4]
```diff
@@ -19,9 +19,17 @@
...
+<!--            auth="Container"-->
+<!--            type="javax.sql.DataSource"-->
+<!--            factory="org.apache.tomcat.jdbc.pool.DataSourceFactory" closeMethod="close"-->
+<!--            minEvictableIdleTimeMillis="60000" timeBetweenEvictionRunsMillis="30000"-->
+<!--            maxWait="20000"-->
+<!--            initialSize="0" minIdle="0" maxIdle="${search.uws.maxActive}" maxActive="${search.uws.maxActive}"-->
+<!--            username="${search.uws.username}" password="${search.uws.password}"-->
+<!--            driverClassName="org.postgresql.Driver" url="${search.uws.url}"-->
+<!--            removeAbandoned="false"-->
+<!--            testOnBorrow="true" validationQuery="select 123" />-->
```
Comment (at88mph) [inline caom2-search-server/examples/default/src/main/webapp/META-INF/context.xml]: Please remove the commented one and add the parameterized values in (`${search.uws.xxx}).  That way it's completely configurable.  In the process, please rename them to `caom2search.uws.xxx`.

**Pull request merged**